templates/site/datacatalog.html

```diff
@@ -23,8 +23,33 @@
     {% with %}
         <script>
             function nodeFilter([_name, data]) {
-                return data.services && data.services.some(service => service.keywords.includes("catalog"))
+                return data.services && data.services.some(service => service.name.includes("stac-browser"))
```

Comment:
includes will look for substrings. Please match the name exactly

Reply:
Changed condition to match name exactly